Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1319,20 +1317,21 @@ namespace Babylon
 
             Napi::Value GetPose(const Napi::CallbackInfo& info)
             {
-                auto* xrSpace = XRReferenceSpace::Unwrap(info[0].As<Napi::Object>());
-                if (xrSpace != nullptr)
+                if (info[0].IsExternal())
```

Comment:
Thanks for the fix Justin this was my intention with the original change, but I had thought that unwrap would just return null in the event the object was an incorrect type.

Reply:
Gotcha. I actually didn't know for sure what it would do either. Turns out it throws a NAPI error associated with invalid arguments, apparently.